### PR TITLE
Add tests for SearchableText indexing in solr and bump ftw.solr.

### PR DESCRIPTION
**Pull request by njohner**
SearchableText is not getting indexed when creating a dossier from a template containing a word document
This is due to the update of the docproperties, which overwrites the blob that will be indexed in Solr in a commit hook. This is fixed in `ftw.solr` (https://github.com/4teamwork/ftw.solr/pull/176), by cleaning up the extract_commands before sending them to solr.

To be able to test whether this fixes the problem in Gever I had to add a new testing layer, for functional tests with solr. This is needed because we never commit in `IntegrationTest`s, leading to blobs not being committed and the `SearchableText` not being indexed in solr.

For For https://4teamwork.atlassian.net/browse/CA-1104

## Checklist (Must have)

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [x] Changelog entry
- [x] Link to issue (Jira or GitHub) and backlink in issue (Jira)

### CODE REVIEW

**Review by lukasgraf — APPROVED** [review]
LGTM 👍 
Just some minor stuff.

File: opengever/testing/helpers.py [hunk 1]
```diff
@@ -223,3 +224,12 @@ def pop_msgs(self):
 
     def clear(self):
         self._records = []
+
+
+class SolrTestMixture(object):
```
[the remark this answers is not in the document]
Comment (njohner) [reply opengever/testing/helpers.py:229]: It's a mix between Mixin and Fixture

File: opengever/core/solr_testing.py [hunk 2]
```diff
@@ -95,9 +95,13 @@ def await_ready(self, timeout=60, interval=0.1, verbose=False):
...
         self._require_configured()
         for index in range(int(timeout / interval)):
-            if self.is_ready():
+            try:
+                ready = self.is_ready()
+            except ConnectionError:
+                ready = False
+            if ready:
                 return self
-            if verbose:
```
[the remark this answers is not in the document]
Comment (njohner) [reply opengever/core/solr_testing.py:100]: oh that's an accident, sorry

File: opengever/core/testing.py [hunk 3]
```diff
@@ -685,6 +684,9 @@ def setUpPloneSite(self, portal):
 
 class OpengeverFixtureWithSolr(SolrTestingBase, OpengeverFixture):
 
+    solr_port = os.environ.get('PORT5', '19905')
+    solr_core = 'testing'
```
[the remark this answers is not in the document]
Comment (njohner) [reply opengever/core/testing.py:688]: Isn't that a separate buildout with a separate solr?

**Pull request merged**